2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_setup.py:_flush():76] Current SDK version is 0.15.12
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_setup.py:_flush():76] Configure stats pid to 4294
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_setup.py:_flush():76] Loading settings from /home/huseyin/.config/wandb/settings
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_setup.py:_flush():76] Loading settings from /home/huseyin/fungtion/dannpy_yeniden/DANN_py3/wandb/settings
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/home/huseyin/fungtion/dannpy_yeniden/DANN_py3/main.py', 'program': 'main.py'}
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_init.py:_log_setup():528] Logging user logs to /home/huseyin/fungtion/dannpy_yeniden/DANN_py3/wandb/run-20240203_102518-ja6dvjez/logs/debug.log
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_init.py:_log_setup():529] Logging internal logs to /home/huseyin/fungtion/dannpy_yeniden/DANN_py3/wandb/run-20240203_102518-ja6dvjez/logs/debug-internal.log
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_init.py:init():568] calling init triggers
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {'ms': 0.05, 'Mt': 0}
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_init.py:init():618] starting backend
2024-02-03 10:25:18,162 INFO    MainThread:4294 [wandb_init.py:init():622] setting up manager
2024-02-03 10:25:18,165 INFO    MainThread:4294 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-03 10:25:18,166 INFO    MainThread:4294 [wandb_init.py:init():628] backend started and connected
2024-02-03 10:25:18,169 INFO    MainThread:4294 [wandb_init.py:init():720] updated telemetry
2024-02-03 10:25:18,177 INFO    MainThread:4294 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-03 10:25:24,325 WARNING MainThread:4294 [wandb_init.py:init():1192] interrupted
Traceback (most recent call last):
  File "/home/huseyin/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1170, in init
    run = wi.init()
  File "/home/huseyin/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 756, in init
    result = run_init_handle.wait(
  File "/home/huseyin/.local/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/huseyin/.local/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/huseyin/.local/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/huseyin/anaconda3/envs/dannpy5/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/huseyin/anaconda3/envs/dannpy5/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
